src/infosquito/icat.clj

```diff
@@ -358,20 +368,36 @@
        "                      WHERE meta_attr_name = 'ipc_UUID' AND meta_attr_value = ?)"))
 
 
-(defn file-exists?
+(defn- exists?*
   [uuid]
   (sql/with-query-results rs [existence-query uuid]
     ((comp pos? :count first) rs)))
 
+(def hitcount (atom 0))
+(def misscount (atom 0))
+(def misscount-t (atom 0))
+(def misscount-f (atom 0))
 
-(defn folder-exists?
+(defn exists?
```

Comment:
Since the logic for `file-exists?` and `folder-exists?` was identical, I combined them.